src/components/profile-card.js

```diff
@@ -1,66 +1,101 @@
-    },
-    {
-      id: 4,
-      name: "최수민",
-      age: 26,
-      univ: "서강",
-      role: "부회장",
-      major: "컴퓨터공학",
-      phoneNum: "010-4413-1261",
-      email: "react@kakao.com",
-      githubLink: "https://github.com/greatSumini",
-      imageUrl:
-        "https://static.wixstatic.com/media/69e8c2_43446e9ac65e4f129083516175259a2b~mv2.jpeg/v1/crop/x_0,y_910,w_3024,h_3122/fill/w_347,h_351,al_c,q_80,usm_0.66_1.00_0.01/%E1%84%8E%E1%85%AC%E1%84%89%E1%85%AE%E1%84%86%E1%85%B5%E1%86%AB.webp"
-    }
-  ];
+export default function ProfileCard (props) {
+  return (
+    <CardWrapper>
+    <ImageContents>
+      <Contents>
+        <Personal>
+          <Name>{props.single.name} ({props.single.age})</Name>
+          <Role>{props.single.role}</Role>
+          <UnivInfo>{props.single.univ}대학교 {props.single.major}과</UnivInfo>
+        </Personal>
+        <Contect>
+          <Phone>{props.single.phoneNum}</Phone>
+          <Email> {props.single.email}</Email>
+          <Git> <a href={props.single.githubLink}>{props.single.githubLink}</a></Git>
+        </Contect>
+      </Contents>
+      <Image><img src={props.single.imageUrl} width="100%"></img></Image>
+    </ImageContents>
+    <Ceos> 신촌 연합 IT 창업 동아리 CEOS</Ceos>
+  </CardWrapper>
+  );
 
-  return <Wrapper>안녕 나는 프로필 카드</Wrapper>;
 }
 
-const Wrapper = styled.div`
-  background-color: red;
-  font-size: 18px;
+//styled-component로 CSS
+
+const Personal = styled.div`
+    display: block;
+
+`
+const Contect = styled.div`
+    display: block;
+
+`
+const ImageContents = styled.div`
+    display: flex;
+    flex-direction: row;
+    -webkit-box-pack: justify;
+    justify-content: space-between;
+    margin-bottom: 1.5rem;
+`
+const Contents = styled.div`
+    display: flex;
+    flex-direction: column;
+    -webkit-box-pack: justify;
+    justify-content: space-between;
+    width: 50%;
+`
+const Name = styled.div`
+    font-size: 2rem;
+    font-weight: bold;
+    margin-bottom: 1rem;
+`
+const Role = styled.div`
+    font-size: 1.4rem;
+    font-weight: bold;
+`
+const UnivInfo = styled.div`
+    font-size: 1.4rem;
+`
+const Phone = styled.div`
+    font-size: 1.4rem;
+    color: rgb(0, 0, 153);
+`
+const Email = styled.div`
+    font-size: 1.4rem;
+    color: rgb(0, 0, 153);
+`
+const Git = styled.div`
+    font-size: 1.4rem;
+`
+const Image = styled.div`
+    width: 40%;
+    height: 40%;
+`
+const Ceos = styled.div`
+    display: flex;
+    flex-direction: column;
+    -webkit-box-align: center;
+    align-items: center;
+    font-size: 1.6rem;
+`
+const CardWrapper = styled.div`
+    width: 45%;
+    background-color: rgb(255, 255, 255);
+    margin-bottom: 2rem;
+    padding: 1.5rem;
+    border-width: 1px;
+    border-style: solid;
+    border-color: rgb(204, 204, 204);
+    border-image: initial;
+    border-radius: 1.5rem;
```

Comment:
코드 라인을 최대한 줄여주시는게 좋아요!
```suggestion
    border: 1px solid rgb(204, 204, 204);
    border-radius: 1.5rem;
```